packages/story-editor/src/components/publishModal/types.js

```diff
@@ -24,6 +24,6 @@ import { INPUT_KEYS } from './constants';
 
 export const MANDATORY_INPUT_VALUE_TYPES = PropTypes.shape({
   [INPUT_KEYS.EXCERPT]: PropTypes.string,
-  [INPUT_KEYS.SLUG]: PropTypes.string,
+  [INPUT_KEYS.SLUG]: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
Wouldn't the type still be a string, e.g. `"5"` and not `5`? Thoughts?

Reply:
I thought so but then i was running through a handful of situations in the editor and it was a number so i updated this here that way we wouldn't get noise in the console.